src/core/src/op/util/op_types.cpp

```diff
@@ -26,60 +26,60 @@
 #include "openvino/op/xor.hpp"
 
 bool ov::op::util::is_unary_elementwise_arithmetic(const ov::Node* node) {
-    return dynamic_cast<const ov::op::util::UnaryElementwiseArithmetic*>(node) != nullptr;
+    return ov::as_type<const ov::op::util::UnaryElementwiseArithmetic>(node) != nullptr;
```

Comment:
Consider to use `is_type` here?

Reply:
In these PRs I focus only on needed changes to avoid review burden.
I agree - `is_type` should be used in such cases.